src/table/components/head/__tests__/HeadCellMenu.spec.tsx

```diff
@@ -264,36 +264,41 @@ describe('<HeadCellMenu />', () => {
       fireEvent.click(screen.getByText(`SNTable.MenuItem.${targetAction}`));
     };
 
-    afterEach(async () => {
```

Comment:
what was wrong with this afterEach ?

Reply:
You cannot have assertions outside of test cases